app/views/vignettes/questionnaires/take.html.erb

```diff
@@ -0,0 +1,69 @@
+
+<% @slide.info_slides.each_with_index do |info_slide, index| %>
+<div id="info-slide-container-<%= index %>" class="info-slide-container" style="display: none;">
+    <h2> Info Slide <%= index + 1 %> </h2>
+    <button class="close-info-slide-button btn btn-primary" data-index="<%= index %>">Close Info Slide</button>
+    <%= info_slide.content %>
+</div>
+<%end%>
+
+<div id="current-slide-container" data-slide-id="<%= @slide.id %>" data-questionnaire-id="<%= @questionnaire.id %>" >
+
+    <h2> Slide <%= @slide.position %> </h2>
+
+    <% @slide.info_slides.each_with_index do |info_slide, index| %>
+        <% if info_slide.icon.attached? %>
+        <button class="info-slide-button info-slide-icon-button" data-index="<%= index %>">
+                <%= image_tag info_slide.icon.variant(:thumb), alt: info_slide.title, class: 'info-slide-icon' %>
+            </button>
+        <% else %>
+            <button class="info-slide-button btn btn-primary" data-index="<%= index %>">
+            <%= info_slide.title %>
+            </button>
+        <% end %>
+    <% end %>
+
+    <%= @slide.content %>
+    <%= @slide.question.question_text %>
+
+    <br>
+
+    <%= form_with(model: @answer, id: "answer_form", url: submit_answer_vignettes_questionnaire_url(@questionnaire), local: true) do |form| %>
+
+
+        <%= form.hidden_field :slide_id, value: @slide.id %>
+
+        <%= form.fields_for :slide_statistic do |stat_form| %>
+            <%= stat_form.hidden_field :user_id, value: current_user.id %>
+            <%= stat_form.hidden_field :time_on_slide, id: "time-on-slide-field" %>
+            <%= stat_form.hidden_field :time_on_info_slides, id: "time-on-info-slides-field" %>
+            <%= stat_form.hidden_field :info_slides_access_count, id:"info-slides-access-count-field" %>
+        <% end %>
+
+        <% case @slide.question.type %> 
+            <% when "Vignettes::TextQuestion" %>
+                <%= form.label :text, "Your Answer" %>
+                <%= form.text_area :text %>
+            <% when "Vignettes::MultipleChoiceQuestion" %>
+                <% @slide.question.options.each do |option| %>
+                    <%= option.text %>
+                    <%= check_box_tag "vignettes_answer[option_ids][]", option.id, @answer.option_ids.include?(option.id) %>
+                <% end %>
+            <% when "Vignettes::LikertScaleQuestion" %>
+                <% Vignettes::LikertScaleQuestion::LIKERT_LABELS.each do |value, label| %>
+                <div>
+                    <%= form.radio_button :likert_scale_value, value %>
+                    <%= form.label :likert_scale_value, label %>
+                </div>
+                <% end %>
+        <% end %>
+        <br>
+        <% if @slide.question.type == "" %>
+            <%= form.submit "Next Slide", class: "btn btn-primary" %>
+        <% else %>
+            <%= form.submit "Submit answer", class: "btn btn-primary" %>
```

Comment:
Bug: It is possible to submit an answer, even though the user has not selected any multiple choice option.